src/EditorFeatures/VisualBasicTest/ConvertLinq/ConvertLinqQueryToLinqMethodTests.vb

```diff
@@ -0,0 +1,117 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.CodeRefactorings
+Imports Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.CodeRefactorings
+Imports Microsoft.CodeAnalysis.VisualBasic.ConvertLinq
+
+Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests.CodeActions.ConvertLinq
+
+    Public Class ConvertLinqQueryToLinqMethodTests
```

Comment:
>ConvertLinqQueryToLinqMethodTests [](start = 17, length = 33)

It looks like there are no tests that cover AsEnumerable, AsQueryable and probably implicit Select for type conversion purposes